gooddata-sdk/tests/catalog/test_catalog_permission.py

```diff
@@ -108,3 +111,42 @@ def test_put_declarative_permissions(test_config):
         _assert_default_permissions(declarative_permissions_o)
     finally:
         _empty_permissions(sdk, workspace_id)
+
+
+@gd_vcr.use_cassette(str(_fixtures_dir / "list_available_assignees.yaml"))
+def test_list_available_assignees(test_config):
```

Comment:
It seems that the test excepts some default behavior. Is it adapted in bootstrap fixtures here?

https://github.com/gooddata/gooddata-python-sdk/tree/master/tests-support/fixtures

Reply:
By my opinion it's not necessary. For available assignees you get available users and user groups to grant permissions for dashboard, so it's clear. For dashboard permissions, I used manage permissions to grant permissions and than remove it with check with call dashboard permissions.